query.py

```diff
@@ -385,7 +385,7 @@ def get_death_causes(c):
       clean_rows.append( [ killer, r[2], r[3] ] )
     last = r
 
-  clean_rows.sort(lambda a,b: int(b[1] - a[1]))
+  clean_rows.sort(key=lambda i:i[1])
```

Comment:
rewritten comparison

Reply:
Again I think you've reversed this one, `a < b` when `b[1] - a[1] < 0`, so this is a reverse sort on `i[1]`